### PR TITLE
feat: added list component and text utilities

### PR DESCRIPTION
**Pull request by sks-21**
Added List component and text utility component

List component includes

- Unordered lists
- Ordered lists
- Basic list
- list with radio buttons
- list with checkbox
- Stacked notification list

Text utility component includes

- H tag headings
- Paragraph heading
- Text alignment
- Text color
- Different font weight heading

### CODE REVIEW

**Review by Omkar-Chaskar — APPROVED** [review]
Looks good to me.

File: components/list/list.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
+@import "/components/component.css";
+
+.list {
+    margin: 1rem;
+}
+
+/* || Basic List  */
+.list-basic-item {
+    padding: 4px;
```
Comment (hussiantabha) [inline components/list/list.css:9]: use rem

File: components/list/list.html [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Futbolt UI | Input</title>
+    <script src="https://kit.fontawesome.com/0a41da8839.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/components/list/list.css" />
+</head>
+
+<body>
+    <!-- || Container  -->
+    <div class="container">
+
```
Comment (Kothalkarpriya) [inline components/list/list.html:16]: remove such extra spaces

**Comment by hussiantabha** [comment]
Looks good to me

File: components/textutils/textutils.html [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Futbolt UI | Input</title>
+    <script src="https://kit.fontawesome.com/0a41da8839.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/components/textutils/textutils.css" />
+</head>
+
+<body>
+    <!-- || Container  -->
+    <div class="container">
+
```
Comment (Kothalkarpriya) [inline components/textutils/textutils.html:16]: here also

**Review by Kothalkarpriya — APPROVED** [review]
Good Work! You can merge, Look into my comments once

**Pull request merged**